pyproject.toml

```diff
@@ -13,7 +13,7 @@ dependencies = []
 dev = [
     "mkdocs<1.7,>=1.6",
     # Pin mkdocs-material to the exact version b/c it sometimes breaks thing
-    "mkdocs-material==9.6.4",
+    "mkdocs-material==9.6.13",
```

Comment:
Updates to mkdocs-material can break things and might noticeably add to the size of the repo if they are done often (since it includes a lot of minified javascript). I'd wait at least until the next release so that we have longer to catch bugs in the docs.

Reply:
Reverted. Note that 9.6.4 has been configured 3 months ago, there have been many bugfixes since then.